src/models/UserModel.ts

```diff
@@ -1 +1,36 @@
-export {}
+import mongoose from 'mongoose'
+import UserInterface from '../interfaces/UserInterface'
+const Schema = mongoose.Schema
+const userSchema = new Schema<UserInterface>({
+  name: {
+    type: String,
+    required: true
+  },
+  email: {
+    type: String,
+    unique: true,
+    required: true
+  },
+  password: {
+    type: String,
+    required: true
+  },
+  slides: [
```

Comment:
You will need to create another model `SlideModel` and use it here like this: 

```js
slides: [Slide.schema],
```

and the slide model will look like this: 
```js
     title: {
        type: String,
        required: true
      },
      link: {
        type: String,
        required: true
      }
```
this way the `_id` should be auto generated.